ts/data/case.ts

```diff
@@ -532,16 +539,73 @@ function addSampleIcons(samples: Sample[], fragment: DocumentFragment) {
   samples.forEach((sample, i) => {
     const status = getSampleQcStatus(sample);
     const icon = makeIcon(status.icon);
-    icon.title =
-      (sample.run ? sample.run.name + " - " : "") +
-      `${sample.name}: ${status.label}`; // TODO: more detailed popup
+    addToolTip(icon, makeSampleTooltip(sample));
     fragment.appendChild(icon);
     if (i < samples.length - 1) {
       fragment.appendChild(document.createTextNode(" "));
     }
   });
 }
 
+function makeSampleTooltip(sample: Sample) {
+  const toolTipConatiner = new DocumentFragment();
+  const topContainer = document.createElement("div");
+  topContainer.className = "flex flex-col space-y-1";
+  const bottomContainer = document.createElement("div");
+  bottomContainer.className =
+    "grid grid-cols-2 grid-flow-row gap-y-1 font-inter font-medium font-14 text-black mt-3";
+  // sample run links
+  if (sample.run) {
+    topContainer.appendChild(
+      makeNameDiv(
+        sample.run.name,
+        urls.miso.run(sample.run.name),
+        urls.dimsum.run(sample.run.name)
+      )
+    );
+  }
+  // sample name links
+  topContainer.appendChild(
+    makeNameDiv(sample.name, urls.miso.sample(sample.id))
+  );
+  // project links
+  const projectLabel = document.createElement("span");
+  projectLabel.innerHTML = "Project:";
+  const projectLink = makeNameDiv(
+    sample.project,
+    urls.miso.project(sample.project),
+    urls.dimsum.project(sample.project)
+  );
+  bottomContainer.appendChild(projectLabel);
```

Comment:
abstract to an `addRow(container, label, name, misoUrl, dimsumUrl)` function that can be reused for the donor name/links below and requisition later on. Add both label and name div to the container in this function